README.rst

```diff
@@ -55,38 +55,42 @@ or
 
 Set up Technical Journal plugin
 +++++++++++++++++++++++++++++++
+The plugin can either be installed directly from pip:
 
-Clone this repository: ``git clone https://github.com/girder/tech_journal``
+.. parsed-literal::
```

Comment:
I'd use the [`code` reStructuredText directive](http://docutils.sourceforge.net/docs/ref/rst/directives.html#code) here. The language can be `bash`.